release notes/v0.44.0.md

```diff
@@ -0,0 +1,221 @@
+    await tails.click(),
+    ```
+
+    A `locator.click` action that does result in a navigation can be used like so:
+    ```javascript
+    const tails = page.locator("input[value='Bet on tails!']");
+    await Promise.all([
+      page.waitForNavigation(),
+      tails.click(),
+    ]);
+    ```
+
+- [browser#817](https://github.com/grafana/xk6-browser/pull/817) We've removed `--no-sandbox` from the default Chrome launch arguments. Now Chrome will launch with a sandbox, which is a more secure way of running the browser. If you are running tests under a `root` user, the browser will no longer launch unless the `--no-sandbox` argument is supplied. You can still pass this flag when launching a new Chrome instance using the `args` parameter on `chromium.launch`:
+
+    ```javascript
+    const browser = chromium.launch({
+      args: ['no-sandbox'],
+    });
+    ```
+
+- [browser#844](https://github.com/grafana/xk6-browser/pull/844) Removed the exported `version` param from the root module. Users should from now on reference the k6 version instead of the browser module version.
+- [browser#838](https://github.com/grafana/xk6-browser/pull/838) Removed first meaningful paint metric. This metric is being deprecated across all the browsers, so we've removed it.
+
+## New features
+
+### A new experimental module implementing the Web Crypto API specification [#2884](https://github.com/grafana/k6/pull/3007)
+
+This release includes a new experiment `webcrypto` module implementing (partially) the [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) specification in k6.
+
+<details>
+<summary> Expand to see an example of the new functionality.</summary>
+
+An example shows encrypting and decrypting of a "Hello, World!" string using AES-CBC algorithm.
+
+```javascript
+import { crypto } from 'k6/experimental/webcrypto';
+
+export default async function () {
+  const key = await crypto.subtle.generateKey(
+    {
+      name: 'AES-CBC',
+      length: 256,
+    },
+    true,
+    ['encrypt', 'decrypt']
+  );
+
+  const encoded = stringToArrayBuffer('Hello, World!');
+  const iv = crypto.getRandomValues(new Uint8Array(16));
+
+  const ciphertext = await crypto.subtle.encrypt(
+    {
+      name: 'AES-CBC',
+      iv: iv,
+    },
+    key,
+    encoded
+  );
+
+  const plaintext = await crypto.subtle.decrypt(
+    {
+      name: 'AES-CBC',
+      iv: iv,
+    },
+    key,
+    ciphertext
+  );
+
+  console.log(
+    'deciphered text == original text: ',
+    arrayBufferToHex(plaintext) === arrayBufferToHex(encoded)
+  );
+}
+
+function arrayBufferToHex(buffer) {
+  return [...new Uint8Array(buffer)].map((x) => x.toString(16).padStart(2, '0')).join('');
+}
+
+function stringToArrayBuffer(str) {
+  var buf = new ArrayBuffer(str.length * 2); // 2 bytes for each char
+  var bufView = new Uint16Array(buf);
+  for (var i = 0, strLen = str.length; i < strLen; i++) {
+    bufView[i] = str.charCodeAt(i);
+  }
+  return buf;
+}
```

Comment:
Unrelated to the release notes, but we should probably expose these as helper functions, along with their reverse counterparts. They seem generic enough to be part of [jslib utils](https://k6.io/docs/javascript-api/jslib/utils/).